scripts/run_backend_tests.py

```diff
@@ -214,7 +215,7 @@ def run(self):
         test_target_flag = '--test_target=%s' % self.test_target
         if self.generate_coverage_report:
             exc_list = [
-                'python', COVERAGE_PATH, 'run', '-p', TEST_RUNNER_PATH,
+                'python', '-m', 'coverage', 'run', '-p', TEST_RUNNER_PATH,
```

Comment:
Is it possible, here and below, to refer to the specific directory, similar to the previous code? It might make troubleshooting easier in the future.

Reply:
I think that is a good idea, as it can also solve issue #8118 .

Reply:
Done!

Reply:
😅I misunderstood what Sean is referring, but what I was thinking is specify a full path for `python`.

Reply:
Hmm, that's another PR right there. :stuck_out_tongue: